src/units/unit.cpp

```diff
@@ -615,6 +627,15 @@ void unit::init(const config& cfg, bool use_traits, const vconfig* vcfg)
 
 	// Adjust the unit's defense, movement, vision, jamming, resistances, and
 	// flying status if this config has its own defined.
+	if(cfg.has_child("movement_costs")
```

Comment:
Maybe `|| cfg.has_attribute("flying")`? (Not 100% sure if this is allowed, check the wiki.)

Reply:
hmm ye that should be added